### DISCUSSION TITLE
Is it safe to include the script kit code in source control (public repo)?

### DISCUSSION THREAD

**Discussion by JwanKhalaf** [FortAwesome/Font-Awesome#20082] [General]
Hello

As the title says. Should I include the kit script code in source control, in other words commit it into a public git repository?

The page where one grabs the kit script code doesn't mention that.

**Comment by robmadole** [comment]
Yep. The Kit code is not a secret or protected piece of data. (It can't be, you can "View source" on a web page and see the URL).

@JwanKhalaf I would encourage you to use our domain filtering feature in Kits. That way only the site you own and use will be able to use the Kit code. (This prevents someone from using your Kit on their site and making you responsible for the pageviews.)
